### PR TITLE
:hammer: Add some notification function

### PR DESCRIPTION
**Pull request by Zeleeer**

### CODE REVIEW

File: src/api/player/client/showAdvancedNotification.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---@param text string
+---@param colorBg number
+---@return void
+---@public
+
+function API_Player:showAdvancedNotification(icon, type, sender, title, text, colorBg)
+    if (colorBg) then
+        ThefeedNextPostBackgroundColor(colorBg)
+    end
+    SetNotificationTextEntry("STRING");
```
Comment (MoskalykA) [inline src/api/player/client/showAdvancedNotification.lua:15]: Why use this ";"?

**Comment by MoskalykA** [comment]
Why use so many files for so few functions?

**Comment by Zeleeer** [comment]
The code format whant one function for one files :)

File: src/api/player/client/showAdvancedNotification.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---showAdvancedNotification
+---@param icon string
+---@param type number
+---@param sender string
+---@param title string
+---@param text string
+---@param colorBg number
+---@return void
+---@public
+
```
Comment (PABLO-1610) [inline src/api/player/client/showAdvancedNotification.lua:10]:
```suggestion

```

File: src/api/player/client/showHelpNotification.lua [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---showHelpNotification
+---@param text string
+---@return void
+---@public
+
```
Comment (PABLO-1610) [inline src/api/player/client/showHelpNotification.lua:5]:
```suggestion

```

File: src/api/player/client/showMissionNotification.lua [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---showMissionNotification
+---@param text string
+---@param time number
+---@return void
+---@public
+
```
Comment (PABLO-1610) [inline src/api/player/client/showMissionNotification.lua:6]:
```suggestion

```

File: src/api/player/client/showNotification.lua [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---showNotification
+---@param text string
+---@param colorBg number
+---@return void
+---@public
+
```
Comment (PABLO-1610) [inline src/api/player/client/showNotification.lua:6]:
```suggestion

```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

**Review by PABLO-1610 — APPROVED** [review]
Thanks for thoses functions 😁

**Pull request merged**